tests/test_geom_operations.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+import numpy as np
+
+import aimsprop as ai
+
+
+def test_compute_bond(trajectory):
+    # Get original trajectory length before computations
+    orig_traj_ln = len(trajectory.frames)
+
+    # Compute the bond distance between atoms 0 and 1 for the trajectory.
+    # Test if the first frame's bond length is correct.
+    # Test if the length of the attribute is the same as the trajectory.
+
+    ai.compute_bond(trajectory, "R01", 0, 1)
+    assert trajectory.frames[0].properties["R01"] == 1.2822983730767554
+    assert len(trajectory.frames) == orig_traj_ln
```

Comment:
What exactly is this testing? We are testing that the number of frames before/after the computation is the same. Is that a critical value we'd expect to change, indicating that something went wrong?

Reply:
Removed as per slack convo :)